src/internal/utils/generateId.ts

```diff
@@ -0,0 +1,4 @@
+import { v4 as uuidv4 } from "uuid";
+
+export const generateId = (type?: string | number) =>
+  type ? `${type}-${uuidv4()}` : uuidv4();
```

Comment:
Why does type matter? We can just use uuidv4 directly can't we?

Reply:
Oh yeah we can, I was just copying puck's method of generateId so it'd match the saveState hash's. I can remove this

Reply:
Yea we don't need type so let's remove